### PR TITLE
Implementing tests

### PR DESCRIPTION
**Pull request by dprokop**

### CODE REVIEW

File: test/utils/combineStates.test.ts [hunk 1]
```diff
@@ -1,22 +1,27 @@
 import { combineStates } from '../../src/utils/combineStates';
 import { StatesType, QuerierState } from '../../src/types';
+import { QueryStateType } from '../../src';
+
+const createStateTypes = (stateTypes: Array<QuerierState>): StatesType => {
```
Comment (dprokop) [inline test/utils/combineStates.test.ts]: @aniaw It's `createStateProps`. I have a task on my agenda to rename Querier's `States` to statuses, as `State` is super easy to confuse with React's `state`

File: package.json [hunk 2]
```diff
@@ -104,6 +109,7 @@
     "prettier": "^1.10.2",
     "prompt": "^1.0.0",
     "react": "^16.2.0",
+    "react-dom": "^16.2.0",
     "redux": "^3.7.2",
     "replace-in-file": "^3.1.1",
     "rimraf": "^2.6.1",
```
Comment (aniaw) [inline package.json:115]: Remove all `^`. Or maybe it is on purpose?

**Comment by aniaw** [comment]
Is a demo from small talk available somewhere?

**Comment by dprokop** [comment]
@aniaw I'm gonna add example early this week. I have planned to do it during weekend but spent it in bed curing flu:/

**Comment by dprokop** [comment]
@aniaw I have quickly pushed the demo to this repo https://github.com/dprokop/querier-ssr-example. Please note, that Querier is not added to dependencies there, so you need to either link your local version (`npm link`) or install from github :) Oh, and BTW you should be at least something familiar with that repository ;)

**Pull request merged**